samples/how-to-deal-with-import/wasm/inc/pthread.h

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
This file already exists in `wamr-sdk/app/libc-builtin-sysroot/include/pthread.h`, we had better only maintain one copy, could we copy it to here in CMakeLists.txt or add `wamr-sdk/app/libc-builtin-sysroot/include` to include dir?

Reply:
Will copy *pthread.h* to `wasm/inc` during configuration phase.

Adding *wamr-sdk/app/libc-builtin-sysroot/include* into headers search paths will leads to conflicts likes:

```
warning: call to undeclared function 'UINT16_C'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
error: use of undeclared identifier 'errno'
```